src/styles/styles/card.scss

```diff
@@ -0,0 +1,125 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  font-family: Roboto, sans-serif;
```

Comment:
It's better to apply font family for the HTML tag to get it inherited by other elements

Reply:
Not fixed